fixtures/uitests/tests/ui/fieldless_errors_used_in_callbacks_cant_have_fields.stderr

```diff
@@ -1,11 +1,13 @@
-error[E0063]: missing field `numerator` in initializer of `ArithmeticError`
-  --> $OUT_DIR[uniffi_uitests]/errors.uniffi.rs
-   |
-   |             2 => r#ArithmeticError::r#DivisionByZero{ },
-   |                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ missing `numerator`
-
-error[E0063]: missing field `0` in initializer of `ArithmeticError`
-  --> $OUT_DIR[uniffi_uitests]/errors.uniffi.rs
-   |
-   |             3 => r#ArithmeticError::r#UnexpectedError{ },
-   |                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ missing `0`
+error[E0533]: expected unit struct, unit variant or constant, found struct variant `Self::DivisionByZero`
+ --> $OUT_DIR[uniffi_uitests]/errors.uniffi.rs
+  |
+  | / ::uniffi::ffi_converter_flat_error_with_read!(
+  | |     r#ArithmeticError
+  | |     {
+  | |         r#IntegerOverflow {
+... |
+  | |     }
+  | | );
+  | |_^
+  |
+  = note: this error originates in the macro `::uniffi::ffi_converter_flat_error_with_read` (in Nightly builds, run with -Z macro-backtrace for more info)
```

Comment:
These error messages aren't very good, but they don't seem any worse that the previous ones.